scripts/appleseed.package.py

```diff
@@ -190,6 +191,103 @@ def merge_tree(src, dst, symlinks=False, ignore=None):
         raise Error(errors)
 
 
+def create_mac_icon(image, icon_path):
+    icon_set = "appleseed.iconset"
+    safe_make_directory(icon_set)
+
+    for size in (16, 32, 64, 128, 256, 1024, 2048):
+        if size < 1024:
+            name = "icon_{0}x{0}.png".format(size)
+            cmd = ['sips', '-z', str(size), str(size), image, '--out', os.path.join(icon_set, name)]
+            # pipe stderr to stdout to silence verbosity
+            subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+        if size > 16:
+            name = "icon_{0}x{0}@2x.png".format(size/2)
+            cmd = ['sips', '-z', str(size), str(size), image, '--out', os.path.join(icon_set, name)]
+            # pipe stderr to stdout to silence verbosity
+            subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+
+    subprocess.check_call(['iconutil', '--convert', 'icns', icon_set, '-o', icon_path])
+    safe_delete_directory(icon_set)
+
+def create_mac_app(staging_dir, app_path, icon_path):
+
+    # cleanup old app
+    safe_delete_directory(app_path)
+
+    contents = os.path.join(app_path, "Contents")
+
+    shutil.move(staging_dir, contents)
+    # shutil.copytree(staging_dir, contents, symlinks=True)
+
+    resources = os.path.join(contents,"Resources")
+    safe_make_directory(resources)
+
+    macos =  os.path.join(contents,"MacOS")
+    safe_make_directory(macos)
+
+    # relative symlink for executable
+    os.symlink('../bin/appleseed-studio', os.path.join(macos, 'appleseed'))
```

Comment:
So the story here is that on macOS we have to rename `appleseed.studio` to `appleseed-studio` in order to allow double-clicking on the file in the Finder to start the app. Is this workaround still necessary?

Reply:
Ah I see, I was wondering why.  I was thinking of just moving the executable too. I was surprised that mac didn't complain about the main executable being a symlink.

Reply:
So, we keep the workaround?